vulnerabilities/models.py

```diff
@@ -198,3 +200,17 @@ def make_data_source(self, batch_size: int, cutoff_date: datetime = None) -> Dat
 
     def __str__(self):
         return self.name
+
+
+class VulnerabilitySeverity(models.Model):
+
+    severity_type = models.CharField(
```

Comment:
See the suggested changes in https://github.com/nexB/vulnerablecode/pull/290/files#r540875185
If these are considered as OK, then this field may become `scoring_system_identifier` and below `severity_value` could become `value`?

Also I would suggest this order for fields and for the unique together:
- "vulnerability"
- "value"
- "scoring_system_identifier"
 -"reference"

Telling this story:
For a "vulnerability", I have a severity "value" computed using a scoring system identified by "scoring_system_identifier" and provided by this "reference"